tests/comfy_cli/cql/test_annotations_source.py

```
@@ -0,0 +1,370 @@
+    started = time.monotonic()
+    with pytest.raises(src.AnnotationFetchError, match="timed out"):
+        src.fetch_pair(deadline=0.2)
+    assert time.monotonic() - started < 5.0
+
+
+def test_fetch_pair_runs_concurrently(monkeypatch):
+    """Both files are in flight at once, so the wait is one timeout, not two."""
+
+    def slow(filename):
+        time.sleep(0.3)
+        return VALID_SUP if filename == src._SUPPORTED_NODES else VALID_DIS
```

Pylint (4.0.6)

[warning] Access to a protected member _SUPPORTED_NODES of a client class

(W0212)